e2echart/test-risk-analysis.html

```diff
@@ -0,0 +1,108 @@
+<html lang="en">
+<head>
+    <title>Risk Analysis</title>
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+    <meta name="description"
+          content="Risk analysis is performed by Sippy to attempt to determine if the failures in this job are
+          abnormal when compared to results for similar jobs over the past week, and amidst on-going incidents
+          in the CI infrastructure. Risk analysis API will not catch everything and is a relatively simple
+          implementation today, please reach out to the Technical Release Team if you spot abnormalities or
+          have suggestions.">
+</head>
+<body onLoad="buildTestCaseTable('#test_case_results')">
+<h1>
+</h1>
+<p>
+    "<a href="TEST_RISK_ANALYSIS_SIPPY_URL_GOES_HERE">Link to Sippy</a>"
```

Comment:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/27508/pull-ci-openshift-origin-master-e2e-aws-ovn-serial/1589660281175281664/artifacts/e2e-aws-ovn-serial/openshift-e2e-test/artifacts/junit/test-risk-analysis.html

I don't think this should be in quotes.  

This also just goes to the main page, will it eventually go somewhere else?

Reply:
There is nothing planned for this. The risk analysis API requires a body. The top page of Sippy is put here just for convenience purposes. I am open for suggestions though.